ext/reflection/php_reflection.stub.php

```diff
@@ -432,6 +432,8 @@ public function getNamespaceName(): string {}
     public function getShortName(): string {}
 
     public function getAttributes(?string $name = null, int $flags = 0): array {}
+
+    public function mayBeCyclic(): bool {}
```

Comment:
+1 for exposing this. This may be beneficial for understanding the GC behaviour as PHP user.